specification/resourcegraph/resource-manager/readme.md

```diff
@@ -40,6 +40,17 @@ semantic-validator: true
 model-validator: true
 message-format: json
 ```
+### Tag: package-2021-03
```

Comment:
Please set this API version to be a default one
tag: package-2021-03  (on line 30)